### DISCUSSION TITLE
TypeError: Cannot read properties of null (reading 'length')

### DISCUSSION THREAD

**Discussion by codingwithsurya** [vercel/next.js#61179] [Help]
### Summary

Trying to startup frontend but receiving TypeError: Cannot read properties of null (reading 'length'). I followed ReadME for https://github.com/DSGT-DLP/Deep-Learning-Playground repo but after running pnpm start -- receiving this TypeError. 

### Additional information

```js
https://gist.github.com/codingwithsurya/ab3358214cba61951e0cd973c04ea102
```

**Comment by codingwithsurya** [comment]
@timneutkens any thoughts?

**Comment by karkir0003** [comment]
@Timer @sokra @huozhi any thoughts?

**Comment by codingwithsurya** [comment]
@wbinnssmith @ijjk Would love some help with this!
